quarkchain/cluster/shard.py

```diff
@@ -401,7 +401,20 @@ def __validate_block_headers(self, block_header_list: List[MinorBlockHeader]):
                 diff = header.difficulty
                 if shard_config.POSW_CONFIG.ENABLED:
                     diff //= shard_config.POSW_CONFIG.DIFF_DIVIDER
-                validate_seal(header, consensus_type, adjusted_diff=diff)
+                with_rotation_stats = False
```

Comment:
1. should use `header.create_time` to compare timestamp
2. make a helper to clean up the code

```python
def __should_use_rotation_stats(header):
  return ...

validate_seal(header, consensus_type, ...., with_rotation_stats=self.__should_use_rotation_stats(header))
```

Reply:
1. I have changed the comparison according to the block height.
2. I have cleaned up the code using the helper. 
Please double-check the code.